build.gradle

```diff
@@ -210,6 +210,27 @@ processTestResources {
 sourceCompatibility = 1.8
 targetCompatibility = 1.8
 
+def createSymlinks(archivePath, symlinkLocation) {
+    exec {
+        commandLine 'ln', '-fs', archivePath, symlinkLocation
+        ignoreExitValue = false
+    }
+}
+
+// Suffix is what will be added to the symlink
+def createAllSymlinks(destinationDir, archivePath, suffix) {
+    def finalSuffix = "-" + suffix
+    if (suffix == "") {
+        finalSuffix = ""
+    }
+
+    def symlinkLocation = destinationDir.toString() + "/hellbender-protected" + finalSuffix + ".jar"
```

Comment:
Can we extract this string literals into constants or perhaps can be deduced from some other existing properities/variables in the Gradle file?